tests/test_keydb.py

```diff
@@ -365,13 +365,20 @@ def test_create_keydb_from_root_metadata(self):
     tuf.keydb.clear_keydb()
 
     # 'keyid' does not match 'rsakey2'.
+    # In this case, the key will be added to the keydb
     keydict[keyid] = rsakey2
 
     # Key with invalid keytype.
     rsakey3 = KEYS[2]
     keyid3 = KEYS[2]['keyid']
     rsakey3['keytype'] = 'bad_keytype'
     keydict[keyid3] = rsakey3
+
+    # New key with a duplicate keyid
+    #rsakey4 = KEYS[1]
+    #keyid4 = KEYS[3]['keyid']
+    #keydict[keyid4] = rsakey4
```

Comment:
As keyids are no longer calculated before use, the client will want a different way to verify that all keys applied to a threshold are unique. I think this may be better left to a separate pr (which I've documented in #1084) for 2 reasons:
* This is a larger change that will require creating a standardized representation for keys
* It is only an issue if the delegating metadata makes a mistake or in the case of multi-role delegations (which are not currently in the reference implementation). Also, this issue [already exists](https://github.com/theupdateframework/tuf/blob/develop/tuf/sig.py#L89) with the current implementation if multiple `keyid_hash_algorithms` are used 

That being said, I have started looking into this issue and could include the fix here if that would be more clear. Otherwise I'll remove this test for now.

Reply:
I'd 

> > Should we have an accompany `securesystemslib` PR to remove `securesystemslib.settings.HASH_ALGORITHMS`?
> 
> Is this setting used by other securesystemslib users, especially intoto? @SantiagoTorres @lukpueh

Doesn't appear in a quick `git grep` of in-toto.